src/components/legend/_legend_item.scss

```
@@ -16,7 +16,7 @@ $legendItemVerticalPadding: $echLegendRowGap / 2;
   }
 
   &:hover {
-    background-color: #e7f0f7;
+    background-color: $euiFocusBackgroundColor;
```

I've been testing the colors and we can change to:

```suggestion
    background-color: $euiColorLightestShade;
```

I was first thinking to remove the background but then it seemed confusing because **hovering all the legend highlights the series**. 

I just think we decided on the wrong color on #820. So let's reserve the `$euiFocusBackgroundColor` for the upcoming `focus` states and use the  `$euiFocusBackgroundColor` when hovering the legend.

---

merged suggestion 0d54de7